src/Projections/Windows.UI.Xaml/Windows.UI.Xaml.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(LibBuildTFMsNoNetStandard)</TargetFrameworks>
+    <Platforms>x64;x86</Platforms>
+    <AssemblyName>Microsoft.Windows.UI.Xaml</AssemblyName>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
+    <ProjectReference Include="..\..\WinRT.Runtime\WinRT.Runtime.csproj" />
+    <ProjectReference Include="..\..\cswinrt\cswinrt.vcxproj" />
+    <InternalsVisibleTo Include="UnitTest" />
+    <ProjectReference Include="..\..\Authoring\WinRT.SourceGenerator\WinRT.SourceGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" SetPlatform="Platform=x64"/>
+    <ProjectReference Include="..\Windows\Windows.csproj" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <CsWinRTFilters>
+-exclude Windows
+-include Windows.UI.Xaml
```

Comment:
Don't we need to also project `Windows.UI.Colors` and exclude the `Windows.UI.Color` addition? The Windows SDK projection doesn't include the `Colors` class as it is part of WUX.

Reply:
What was the reason again `Windows.UI.Color` is not projected in the Windows SDK too?

Reply:
The Color struct is projected, but not the Colors class as it is WUX, that's why WinUI 3 has its own implementation of Colors.

Reply:
No I know that but I mean why does WinUI 3 have its own implementation, doesn't the `Color` type and the `Colors` type not have any dependencies on UWP anyway? I mean why couldn't WinUI 3 just reuse `Colors` too in the same way it's also reusing `Color` anyway? Am I missing something?

Reply:
The `Windows.UI.Colors` class is implemented in WUX DLL, and so to use it you would have to load WUX, which I assume is unideal.

![Screenshot of Windows.UI.Colors in OleViewDotNet](https://github.com/microsoft/CsWinRT/assets/29563098/c56b4bda-310a-45c2-94b7-05a275fd4bea)

Reply:
That was probably one of the reasons.  The other is probably that they want the capability to easily add to the Colors static class in an undocked way if they wanted without touching the system one.  Whereas the Color struct is most likely not going to change.

Reply:
Synced with Mano offline, we can revisit this and fix this in a follow up PR, resolving for now.